export class AuthNotFoundException extends Error {
  constructor(id: string) {
    super(`Autenticaci√≥n con ID ${id} no encontrada`);
    this.name = 'AuthNotFoundException';
  }
} 